docker-compose.yml

```diff
@@ -52,6 +51,6 @@ services:
 
 networks:
   internal_network:
+    driver: bridge
 volumes:
-  postgres:
-    name: db_data
\ No newline at end of file
+  postgres:
```

Comment:
No db name?

Reply:
apparently specifying a dbname might lead to issues? GPT said, so I'm not sure. It recommended to remove